validation/Program.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using YamlDotNet.Serialization;
+
+namespace AdvocateValidation
+{
+    class Program
+    {
+
+        readonly static string _advocatesPath =
+#if DEBUG
+            Path.Combine("../../../../", "advocates");
+#else
+            Path.Combine("../", "advocates");
+#endif
+
+        readonly static IDeserializer _yamlDeserializer = new DeserializerBuilder().Build();
+
+        static async Task Main(string[] args)
+        {
+            var advocateList = new List<CloudAdvocateYamlModel>();
+
+            var advocateFiles = Directory.GetFiles(_advocatesPath);
+
+            await foreach (var (path, advocate) in GetAdvocateYmlFiles(advocateFiles).ConfigureAwait(false))
+            {
+                #region Uncomment to enable GitHub Url Validation
+                //var gitHubUrl = advocate.Connect.FirstOrDefault(x => x.Title.Equals("GitHub", StringComparison.OrdinalIgnoreCase))?.Url;
+
+                //if (gitHubUrl is null)
+                //    throw new Exception($"Missing GitHub Url: {path}");
+
+                //if (!gitHubUrl.IsWellFormedOriginalString())
+                //    throw new Exception($"Invalid GitHub Url: {path}");
+                #endregion
```

Comment:
To enable GitHub Url validation, uncomment this code.

It is current commented, because certain users, e.g. Penina Weiss, are missing their GitHub information